data/sheragi/archaeology missions.txt

```diff
@@ -1289,42 +1289,49 @@ mission "Sheragi Archaeology: The Emerald Sword 3"
+			`	After many long hours walking around, the lights of the ship finally flicker to life, along with sirens bringing attention to every other damaged subsystem, startling everyone. One of the specialized crew members comes to you, explaining that they finally managed to turn on the power plant and fix enough of the ship to move it around, but that they need to land it in order to bring in the heavier equipment and fix the rest of it.`
 
 
 
-mission "Sheragi Archaeology: Emerald Sword Epilogue"
+mission "Sheragi Archaeology: Epilogue"
 	landing
 	name "The Emerald Sword"
 	description "Return to Zug."
-	source "Echoes of Tragedy"
+	source "Valley of the Damned"
 	destination "Zug"
-	passengers 20
 	cargo "Outfitting Equipment" 25
-	
 	to offer
 		has "Sheragi Archaeology: The Emerald Sword 3: done"
-	
+	on offer
+		give ship "Emerald Sword (Fixed)" "Emerald Sword"
+		give ship "Black Diamond" "Discount Flak Round"
+		give ship "Black Diamond" "How Do I Steer"
+		give ship "Black Diamond" "An Implicit Comparison To Fleas"
+		give ship "Black Diamond" "Punch Biopsy"
+		give ship "Black Diamond" "Just The Letter H"
+		give ship "Black Diamond" "I Got The Bees You Need"
+		conversation
+			`As you enter the atmosphere of the alien world, you begin running some scans of the air quality to determine if it is safe to breathe. You quickly learn why it is that the Sheragi perished in space; the atmosphere has a high concentration of chlorine gas, fatal to all living creatures aside from those that seem to have miraculously evolved here. It takes some time to find a suitable landing spot - high enough to escape the highest concentrations of chlorine gas in the atmosphere, but flat enough to land on. As luck would have it, the shipboard scanner reveal anomalous structures over a mesa, and as you come in for a landing, it becomes apparent that these are the ruins of a large settlement. The environmental suits brought along "just in case" turn out to be very useful here. The atmosphere would be unhealthy to breathe for long periods of time, and very uncomfortable in the short term`
+			`	While the crew starts to work on the Emerald Sword, you and the archaeologists head into the settlement, documenting what they see. The buildings were all connected to each other to avoid the exterior atmosphere, although nowadays, not much remains. They were made of glass, scrap, and a kind of yellowish white plastic, the same plastic that the carapaces of animals here are made of. They were melted into shape and built into walls over time, giving the buildings a rounded, anachronistic look, like mud huts made of modern materials. All around the inside there are curtains and rags hanging from the ceiling to the ground. Closer inspection reveal those rags to have traces of chemicals, likely made to react with the chlorine in the air to keep the atmosphere inside breathable.`
```

Comment:
```suggestion
			`	While the crew starts to work on the Emerald Sword, you and the archaeologists head into the settlement, documenting what you see. The buildings were all connected to each other to avoid the exterior atmosphere, although nowadays, not much remains. They were made of glass, scrap, and a kind of yellowish white plastic, the same plastic that the carapaces of animals here are made of. They were melted into shape and built into walls over time, giving the buildings a rounded, anachronistic look, like mud huts made of modern materials. All around the inside there are curtains and rags hanging from the ceiling to the ground. Closer inspection reveal those rags to have traces of chemicals, likely made to react with the chlorine in the air to keep the atmosphere inside breathable.`
```